website/docs/includedirsafter.md

```diff
@@ -0,0 +1,39 @@
+Specifies the include directories to parse last per the toolset ordering and marks the directory as an external include directory.  If the exporter or toolset
+does not support include directory ordering, these directories are added to the external include directory path.
+
+```lua
+includedirsafter { "paths" }
+```
+
+### Parameters ###
+
+`paths` specifies a list of include file search directories. Paths should be specified relative to the currently running script file.  Search order is evaluated from
+right to left.
```

Comment:
According to https://gcc.gnu.org/onlinedocs/gcc/Directory-Options.html
it is left-to-right (for gcc at least).

Reply:
Good catch. I'm pretty sure it was in my head correct, I just wrote it down wrong.